common/app/uiComponent/core/RenderingActor.scala

```diff
@@ -0,0 +1,23 @@
+package uiComponent.core
+
+import akka.actor.Actor
+import model.ApplicationContext
+import uiComponent.UIComponent
+import scala.util.Try
+
+case class Rendering(component: UIComponent, ac: ApplicationContext)
```

Comment:
I think often the available messages for an actor go in a companion object, but in this case I suppose it may be cleaner like this.